spec/factories/participant_identity.rb

```diff
@@ -10,12 +10,5 @@
     trait :npq_origin do
       origin { "npq" }
     end
-
-    initialize_with do
-      ParticipantIdentity.find_or_initialize_by(email: email) do |identity|
-        identity.user = user
-        identity.external_identifier = user.id
-      end
-    end
```

Comment:
This now uses the service class in the participant_profile factory. Also removes the need to maintain the same logic in the production code and test code which further enhance the class' test coverage.
We could be quite aggressive and delete `spec/services/identity/create_spec.rb` since it this class would be used all over the place by the `:participant_profile` factory.